Jenkinsfile

```diff
@@ -57,6 +59,8 @@ pipeline {
             deleteDir()
             gitCheckout(basedir: "${BASE_DIR}", githubNotifyFirstTimeContributor: true, shallow: false,
                         reference: '/var/lib/jenkins/.git-references/apm-agent-java.git')
+            // Prepare the maven opentelemetry extension
+            prepareMavenExtension()
```

Comment:
Once we release a version of https://github.com/elastic/opentelemetry-maven-extension then we will change this particular implementation. 

For the time being this will help to test the maven otel extension in this project.